common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -58,7 +39,23 @@
         % endfor
         <span id="answer_${id}"></span>
     </fieldset>
-
+    <div class="indicator_container">
+        % if input_type == 'checkbox' or not value:
+            <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}"
+                id="status_${id}"
```

Comment:
Are there line breaks here due to a max line length, or for visual formatting? I'd prefer it if these were on one line if possible.